qiskit_experiments/calibration_management/base_calibration_experiment.py

```diff
@@ -44,54 +44,53 @@ class BaseCalibrationExperiment(BaseExperiment, ABC):
 
     This abstract class extends a characterization experiment by turning it into a
     calibration experiment. Such experiments allow schedule management and updating of an
-    instance of :class:`.Calibrations`. Furthermore, calibration experiments also specify
-    an auto_update variable which, by default, is set to True. If this variable,
-    is True then the run method of the experiment will call :meth:`~.ExperimentData.block_for_results`
-    and update the calibrations instance once the backend has returned the data.
+    instance of :class:`~qiskit_experiments.calibration_management.Calibrations`. Furthermore, calibration experiments also specify
+    an `auto_update` variable which, by default, is set to True. If this variable
+    is True, then the `run` method of the experiment will call :meth:`~qiskit_experiments.base_experiment.ExperimentData.block_for_results`
+    and update the `Calibrations` instance once the backend has returned the data.
 
-    This mixin class inherits from the :class:`.BaseExperiment` class since calibration
-    experiments by default call :meth:`~.ExperimentData.block_for_results`. This ensures that the next
+    This mixin class inherits from the :class:`~qiskit_experiments.base_experiment.BaseExperiment` class since calibration
+    experiments by default call :meth:`~qiskit_experiments.base_experiment.ExperimentData.block_for_results`. This ensures that the next
     calibration experiment cannot proceed before the calibration parameters have been
     updated. Developers that wish to create a calibration experiment must subclass this
     base class and the characterization experiment. Therefore, developers that use this
     mixin class must pay special attention to their class definition. Indeed, the first
     class should be this mixin and the second class should be the characterization
-    experiment since the run method from the mixin must be used. For example, the rough
-    frequency calibration experiment is defined as
+    experiment since the `run` method from the mixin must be used. For example, the rough
+    frequency calibration experiment is defined as follows:
 
     .. code-block:: python
 
-        RoughFrequencyCal(BaseCalibrationExperiment, QubitSpectroscopy)
+        class RoughFrequency(BaseCalibrationExperiment, QubitSpectroscopy):
```

Comment:
This should have `Cal` at the end. We use this as convention to indicate that it is a calibration experiment.
```suggestion
        class RoughFrequencyCal(BaseCalibrationExperiment, QubitSpectroscopy):
```